iij/ngx_http_secure_token_iijpta.c

```diff
@@ -0,0 +1,191 @@
+#include "ngx_http_secure_token_iijpta.h"
+#include "../ngx_http_secure_token_filter_module.h"
+#include "../ngx_http_secure_token_utils.h"
+
+#include <time.h>
+#include <openssl/hmac.h>
+#include <openssl/evp.h>
+
+// constants
+#define CRC32_SIZE    4
+#define DEADLINE_SIZE 8
```

Comment:
instead of CRC32_SIZE & DEADLINE_SIZE, it will probably make nicer code to define a struct - 
```
typedef struct {
  u_char size[4];
  u_char expiry[8];
} ngx_http_secure_token_iijpta_header_t;
```
(I also prefer the name `expiry` over `deadline`)

Reply:
fix at a27d59a